.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you please remove this file from the PR?

Reply:
Sure, removed.